### ISSUE TITLE
UX Design: Improve contrast in form fields in Django Admin

### ISSUE THREAD

**Issue by PaulKuykendall** [cisagov/manage.get.gov#1522]
### Issue description

One of our UX research subjects noted that the boxes for the form fields were too light and difficult to see.

Review and redesign as appropriate to improve the usability in these fields.

### Acceptance criteria

- [x] Design ~~and Dev collaborate to~~ makes decisions regarding optimal colors and contrast
- [x] Dev ticket created with clear changes to be made, if necessary. ~~(work may completed as we go)~~

### Additional context

Note from [usability study affinity mapping](https://miro.com/app/board/uXjVNDibjyQ=/?openComment=3458764573276489328
)

### Links to other issues

- Relates to [Admin: Copy button has poor color contrast on hover](https://github.com/cisagov/manage.get.gov/issues/2214)
- Relates to [Admin: Sidenav selected item text contrasts poorly in dark mode](https://github.com/cisagov/manage.get.gov/issues/2213)

### Prioritization Comments
**WCAG Conformance Level:** 
**WCAG Standard that applies:** 
**Critical Failure if Not Met?:**  
**Prioritization:** 
**Reason:**

**Comment by abroddrick** [comment]
@PaulKuykendall @Katherine-Osos or @AnnaGingle it seems like this would apply to both light and dark mode correct? Should we hold off on all contrast/dark mode tickets until this one is completed? Also is there a design/product priority here for this one as clearly design time will be needed for this

**Comment by AnnaGingle** [comment]
Our checkbox styling is applied via browser settings. Notice how mine are pink?
<img width="603" alt="Screenshot 2024-09-26 at 2 03 59 PM" src="https://github.com/user-attachments/assets/fd51ee27-5a2c-4fd5-be5e-27d9d75da4aa">


I don't think this would apply to dark mode.
<img width="725" alt="Screenshot 2024-09-26 at 2 02 02 PM" src="https://github.com/user-attachments/assets/c1afc958-9d7f-4b4f-b80d-8fac4fe657d8">

**Comment by AnnaGingle** [comment]
@PaulKuykendall Or could this ticket be about these checkboxes? Do you know which ones the participant was talking about?
<img width="149" alt="Screenshot 2024-09-26 at 2 05 11 PM" src="https://github.com/user-attachments/assets/f5cdf743-7fa3-4a73-87da-aa45e4a21522">

**Comment by h-m-f-t** [comment]
I'm pretty close to calling this "closed as not planned". This was awhile ago and I don't know that we have much to go on.

**Comment by AnnaGingle** [comment]
Can confirm the table checkboxes are using system settings styles. cc @abroddrick. 

The green and black checkboxes meet contrasts standards.

**Comment by SamiyahKey** [comment]
I double-checked this issue, the contrast for those form field boxes are currently not meeting [WCAG 1.4.3 Contrast (Minimum)](https://www.w3.org/TR/WCAG20/#visual-audio-contrast-contrast)

At minimum contrast should be 4.1:1 for text and 3:1 for UI components. ![Image](https://github.com/user-attachments/assets/a606f75a-4cd5-4852-b11e-996f29c9d4dd)
Here is a color picker contrast test for this component:

**Comment by abroddrick** [comment]
I think this is a good ticket to have a discussion about, I think we should look at contrast on django admin, but it would take a bit over 2 days. We have several tickets in our backlog that mention contrast issues on django admin and we should just tackle these at the same time.

**Comment by witha-c** [comment]
Including this as a design/dev collab in one sprint is too much for the coming sprint, so let's focus this ticket on design and putting together a solution then creating a dev ticket for implementation. Design will still ask dev questions, but this won't include implementation.  CC: @Katherine-Osos

**Comment by h-m-f-t** [comment]
Are we inheriting this contrast from Django Admin? If so, can we determine the fix and file a PR on https://github.com/django/django so it's fixed for everyone?

**Comment by allly-b** [comment]
@kimmsie was the completeion of this ticket soley to make #3126 ? We have a couple  other contrast tickets for django admin, so I'm a bit surprised only one came out of this (though maybe it's because the design portion was limited to form fields only?)

**Comment by kimmsie** [comment]
@allly-b Yes - it was specific to the form field boxes based on someone's comment from a usability test. This is the only contrast ticket I worked on for django admin.
